HVTranslate/HentaiVerse汉化.user.js

```diff
@@ -2839,28 +2837,33 @@ var words = {
 
     // function to decide whether a parent tag will have its text replaced or not
     function isTagOk(tag) {
-        return tagsWhitelist.indexOf(tag) === -1;
+        return !tagsWhitelist.includes(tag);
     }
 
     //翻译用到的字典变量
-    var regexps = {}; //存储转换过的字典，key值和word字典对应分组名相同，value格式见下方buildDict;
+    const regexps = new Map(); //存储转换过的字典，key值和word字典对应分组名相同，value格式见下方buildDict;
```

Comment:
`Map` performs better in scenarios involving frequent additions and removals of key-value pairs.